whois-commons/src/main/java/net/ripe/db/whois/common/override/OverrideCredentialValidator.java

```diff
@@ -30,7 +30,12 @@ public boolean isAllowedAndValid(final boolean isTrusted, final UserSession user
             return false;
         }
         final OverrideCredential.OverrideValues overrideValues = overrideCredential.getOverrideValues().get();
-        return isAllowedToUseOverride(isTrusted, userSession, overrideValues.getUsername()) && isValidOverride(overrideValues, objectType);
+
+        try {
+            return isAllowedToUseOverride(isTrusted, userSession, overrideValues.getUsername()) && isValidOverride(overrideValues, objectType);
+        } catch (EmptyResultDataAccessException e){
```

Comment:
the try catch for the update (which is directly calling `isValidOverride` ) is different
so we need to keep this catch here, for the lookups